Wikipedia/UI-V5/WMFArticlePreviewCell.m

```diff
@@ -24,6 +30,18 @@ - (void)prepareForReuse {
     self.summaryLabel.text = nil;
 }
 
+-(void)awakeFromNib {
+    [self rememberSettingsFromIB];
+}
+
+-(void)rememberSettingsFromIB {
+    NSRange r = NSMakeRange(0, 1);
+    self.summaryLabelAttributesFromIB = [self.summaryLabel.attributedText attributesAtIndex:0 effectiveRange:&r];
```

Comment:
We should just do this in code. It's a bit tacky to configure this in IB and then grab it. If you really want to see what it looks like in IB, then we should implement the IBDesignable methods to make that happen.